app/models/ont/library_request.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Ont
+  # Ont::LibraryRequest
+  class LibraryRequest < ApplicationRecord
+    belongs_to :library, foreign_key: :ont_library_id,
+                         inverse_of: :library_requests, dependent: :destroy
+    belongs_to :request, foreign_key: :ont_request_id, inverse_of: :library_requests
+    belongs_to :tag, foreign_key: :tag_id, inverse_of: false
```

Comment:
What does `inverse_of: false` do?

Reply:
I'm guessing It specifies that the model to which this belongs does not need to define an inverse. I took this from some code elsewhere in the app. Will see if I can remove it

Reply:
Rubocop complains if an inverse_of isn't included 😞 _app/models/ont/library_request.rb:9:5: C: Rails/InverseOf: Specify an :inverse_of option._